sql/core/src/test/scala/org/apache/spark/sql/TPCDSQuerySuite.scala

```diff
@@ -82,13 +82,17 @@ class TPCDSQuerySuite extends BenchmarkQueryTest with TPCDSSchema {
     "q3", "q7", "q10", "q19", "q27", "q34", "q42", "q43", "q46", "q52", "q53", "q55", "q59",
     "q63", "q65", "q68", "q73", "q79", "q89", "q98", "ss_max")
 
+  // List up the known queries having too large code in a generated function
+  val blackListForMethodCodeSizeCheck = Set("modified-q3")
```

Comment:
Can we have an IDed TODO to remove this, @maropu ? [SPARK-29128](#25827) is the one right?

Reply:
Otherwise, we had better wait for the fix.

Reply:
ok